lib/templates/src/App.vue

```diff
@@ -32,6 +32,13 @@
         <button @click="$exp.nextScreen">Begin the experiment</button>
       </Screen>
 
+      <Screen :title="'Welcome to the Interactive Experiment'">
+        <template #0>
+          <InteractiveWelcome></InteractiveWelcome>
+          <button @click="$exp.nextScreen">Continue to the lobby</button>
+        </template>
+      </Screen>
```

Comment:
Does it make sense to combine InteractiveWelcome and Lobby into one custom screen? That way the user can click through the explanation and consent delcaration and then we have one wait screen that both initializes the websocket and potentially waits for other group members. (This could be done with screen slides, where the next slide is called once the experiment is set up)

Reply:
Sure, this would make sense and I can try it. The last time I tried I had some difficulties syncing the states in Vuex since the callbacks happened too fast, but I think that should definitely be solvable. I'll take a look at how slides work in general.